src/JSON For mIRC.mrc

```diff
@@ -1819,7 +1819,7 @@ alias -l jfm_SaveDebug {
 
   ;; if a file was specified and it either doesn't exist or the user wants to overwrite the file
   ;;    save the debug buffer to the specified file
-  if (%File) && (!$isfile(%File) || $input(Are you sure you want to overwrite $nopath(%File), qysa, @SReject/JSONForMirc/Log, Overwrite)) {
+  if (%File) && (!$isfile(%File)) || ($input(Are you sure you want to overwrite $nopath(%File), qysa, @SReject/JSONForMirc/Log, Overwrite)) {
```

Comment:
This is incorrect. The correct version would be the following but I choose not to include the extra `()`:
```
if (%File) && ((!$isfile(%File)) || ($input(Are you sure you want to overwrite $nopath(%File), qysa, @SReject/JSONForMirc/Log, Overwrite))) {
```